part-spec/power.json

```diff
@@ -697,6 +697,516 @@
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentTyp": {
+          "description": "typical charging current of a device in charge termination phase",
+          "comment": "units of amps",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentMin": {
+          "description": "minimum charging current of a device in charge termination phase",
+          "comment": "units of volts",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentMax": {
+          "description": "maximum charging current of a device in charge termination phase",
+          "comment": "units of amps",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentAccuracyTyp": {
+          "description": "typical termination charging current regulation accuracy of a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentAccuracyMin": {
+          "description": "minimum termination charging current regulation accuracy of a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentAccuracyMax": {
+          "description": "maximum termination charging current regulation accuracy of a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryChargeVoltageTyp": {
+          "description": "typical battery charge voltage regulated by a device",
+          "comment": "units of volts",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryChargeVoltageMin": {
+          "description": "minimum battery charge voltage regulated by a device",
+          "comment": "units of volts",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryChargeVoltageMax": {
+          "description": "maximum battery charge voltage regulated by a device",
+          "comment": "units of volts",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryChargeVoltageAccuracyTyp": {
+          "description": "typical accuracy of battery charge voltage regulated by a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryChargeVoltageAccuracyMin": {
+          "description": "minimum accuracy of battery charge voltage regulated by a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryChargeVoltageAccuracyMax": {
+          "description": "maximum accuracy of battery charge voltage regulated by a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "efficiency": {
+          "description": "charge efficiency vs charge current of a device",
```

Comment:
this might be a series of graphs depending on Vin/Vout

Reply:
I think it's ok because the series of graph option should already be covered by the way we defined "graph".